src/index.ts

```diff
@@ -94,6 +143,14 @@ interface Route {
 	try(path: string): string | undefined
 }
 
+interface Resp extends ServerResponse {
```

Comment:
Name your variables and types normally.
```suggestion
interface Response extends ServerResponse {
```